src/lando/main/scm/git.py

```diff
@@ -177,7 +180,55 @@ def apply_patch(
             ]
 
             for c in cmds:
-                self._git_run(*c, cwd=self.path)
+                try:
+                    self._git_run(*c, cwd=self.path)
+                except SCMException as exc:
+                    if "error: patch" in exc.err:
+                        raise PatchConflict(exc.err) from exc
+
+    def process_merge_conflict(
+        self,
+        exception: PatchConflict,
```

Comment:
nit: I think it would be nice to rename this argument to something else other than `exception`.

Reply:
Yep. I found the passing the exception a bit iffy anyway. Turns out I can just extract the error message from the Exception in the landing worker, and pass a clean string here instead.